src/utils/tekst.ts

```diff
@@ -0,0 +1,5 @@
+export const ordetNavIStoreBokstaverSkalKunStarteMedStorBokstav = (
```

Comment:
```suggestion
/*
 * Skal endre alle forekomster av NAV med store bokstaver til Nav med kun stor forbokstav
 * */
export const mapNAVtilNav = (
```

Synes metodenavnet ble for verbost :p